docs/hub-deployment-guide/new-cluster/aws.md

```diff
@@ -152,37 +156,61 @@ enough permissions for automatic deployment of hubs from CI/CD. Since these
 credentials are checked-in to our git repository and made public, they should
 have least amount of permissions possible.
 
-1. Fetch credentials we can encrypt and check-in to our repository so
-   they are accessible from our automatic deployment.
+1. First, make sure you are in the right terraform directory:
+
+   ```bash
+   cd terraform/aws
+   ```
+
+1. Fetch credentials for automatic deployment
 
    ```bash
    terraform output -raw continuous_deployer_creds > ../../config/clusters/<your-cluster-name>/deployer-credentials.secret.json
-   cd ../..  # sops commands must be run from the root of the repo
-   sops --output config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json --encrypt config/clusters/<your-cluster-name>/deployer-credentials.secret.json
    ```
 
-   Double check to make sure that the `config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json` file is
-   actually encrypted by `sops` before checking it in to the git repo. Otherwise
-   this can be a serious security leak!
+1. Encrypt the file storing the credentials
 
-2. Grant the freshly created IAM user access to the kubernetes cluster. As this requires
-   passing in some parameters that match the created cluster, we have a `terraform output`
-   that can give you the exact command to run.
+   ```bash
+   sops --output ../../config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json --encrypt ../../config/clusters/<your-cluster-name>/deployer-credentials.secret.json
+   ```
+
+1. Double check to make sure that the `config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json` file is
+   actually encrypted by `sops` before checking it in to the git repo. Otherwise this can be a serious security leak!
+
+    ```bash
+   cat ../../config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json
+   ```
+
+1. Delete the unencrypted file and check the encrypted credentials into our repository so they are accessible from our automatic deployment.
```

Comment:
A shorthand way of removing non-encrypted files (and anything gitignored) is to run `git clean -Xfd`. See https://infrastructure.2i2c.org/en/latest/topic/access-creds/secrets.html#cleaning-up-decrypted-files

Reply:
Good point. I removed the deletion step entirely, thank you!